### PR TITLE
Build for macOS and Linux with prebuilt sync binaries

### PR DESCRIPTION
**Pull request by fealebenpae**

### CODE REVIEW

File: wrappers/Dockerfile.xenial [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM ubuntu:xenial
```
Comment (nirinchev) [inline wrappers/Dockerfile.xenial]: I may be missing something, but looks like this is unused?

File: wrappers/build.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```
Comment (nirinchev) [inline wrappers/build.sh]: Would be nice to update the readme on how to build.

**Comment by nirinchev** [comment]
If you rebase on master, you should get the fixes for the sporadic CI issues

**Comment by fealebenpae** [comment]
> do you think it makes sense to run tests on other linux distros just in case?

We don't do that in JS, but I think it might. I wanted Debian 8 because that's what the Microsoft dotnet Docker images are based on, but we ought to add others as soon as we decide on a subset of distros we'll officially support.

File: wrappers/Dockerfile.xenial (hunk 1 above)
Comment (fealebenpae) [inline wrappers/Dockerfile.xenial]: It is unused, but I wanted to keep it in case we need it again - it was a pain to set up.

**Pull request merged**